Assets/Scripts/Core/Services/Updater/IProjectUpdater.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Core.Services.Updater
+{
+    public interface IProjectUpdater
+    {
+        event Action UpdateCalled;
+        event Action FixedUpdateCalled;
+        event Action LateUpdateCalled;
```

Comment:
Event 'LateUpdateCalled' is never subscribed to

Reply:
that is for future :)